client/components/AgendaModal.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { CreateEditAgendaForm } from '../containers'
+import { ModalWithForm } from './index'
+
+/**
+* Modal for adding and editing an agenda
+*/
+export const AgendaModal = ({ handleHide, modalProps }) => {
+    if (modalProps && modalProps.agenda) {
+        const { agenda } = modalProps
+
+        return (
+            <ModalWithForm
+                title="Edit an Agenda"
+                onHide={handleHide}
+                form={CreateEditAgendaForm}
+                show={true}
+                initialValues={agenda}/>
+        )
+    } else {
+        return (
+            <ModalWithForm
```

Comment:
Again, there is maybe no need to duplicate this component instantiation. We can maybe just ues the condition for the `title` and `initialValues` props.